rskj-core/src/main/java/co/rsk/peg/lockingcap/constants/LockingCapRegTestConstants.java

```diff
@@ -0,0 +1,33 @@
+package co.rsk.peg.lockingcap.constants;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.vote.AddressBasedAuthorizer;
+import java.util.Collections;
+import java.util.List;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+
+public class LockingCapRegTestConstants extends LockingCapConstants {
+
+    private static final LockingCapRegTestConstants instance = new LockingCapRegTestConstants();
+
+    private LockingCapRegTestConstants() {
+        ECKey authorizerPublicKey = ECKey.fromPublicOnly(Hex.decode(
```

Comment:
check that here you named it `authorizerPublicKey` and in [mainnet](https://github.com/rsksmart/rskj/pull/2589/files#diff-7eb4e14993a959654a6715a0c76922524bc297d4116994ad9c15ce6bc40e383bR16) you named it `increaseAuthorizedKeys`. Lets standardize it

Reply:
Yes, this is because, we are dealing with only one key in `RegTest`, unlike `MainNet` and `TestNet`. We don't need to deal with it like a list of keys. For this reason, I deal with it differently. Nevertheless, after this variable, you'll find the `increaseAuthorizedKeys`.

Reply:
No big deal, but I would still call it `increaseAuthorizedKeys`

The variable name shouldn't change depending on the amount of elements. It's still a list, with one, zero or many elements

Reply:
@marcos-iov, The name `increaseAuthorizedKeys` remains. I didn't change that variable. I only changed how we deal with the public key. Actually, it is a suggestion by @antonio and you in Whitelist where we deal with one public key as well and not a list of keys. I used the same variable name indeed.  Please take a look at [Whitelist code](https://github.com/rsksmart/rskj/blob/5be021fbb3ba0a11f96160de57ad3437e9acac5f/rskj-core/src/main/java/co/rsk/peg/whitelist/constants/WhitelistMainNetConstants.java#L17)

Reply:
Ok, got it. Thanks